src/vm/peimage.cpp

```diff
@@ -969,7 +969,7 @@ PTR_PEImageLayout PEImage::CreateLayoutMapped()
     {
         // For CoreCLR, try to load all files via LoadLibrary first. If LoadLibrary did not work, retry using 
```

Comment:
>  retry using regular mapping - but not for native images.

This retry logic won't work for IJW images. If we happen to load IJW image using this retry logic, we need to reject it and fail the load.

Reply:
Added a throw if this doesn't work, but loading with bNTSafeLoad=TRUE does for IJW images